components/raftstore/src/store/fsm/apply.rs

```diff
@@ -958,21 +974,22 @@ where
         if !data.is_empty() {
             let cmd = util::parse_data_at(data, index, &self.tag);
 
-            if apply_ctx.yield_high_latency_operation && has_high_latency_operation(&cmd) {
+            if has_high_latency_operation(&cmd) && apply_ctx.yield_high_latency_operation {
```

Comment:
Seems the former logic is more efficient if `yield_high_latency_operation` is false.